display-cat-food.js

```diff
@@ -0,0 +1,43 @@
+function showCatFoods (catFoods) {
+
+  var catFoodBrand = document.getElementById("brand-name");
+
+  var numBrands = catFoods.cat_brands.length;
+
+  var catfoodTable = document.getElementById("cat-food-table");
+
+  var cumTable = "";
+  
+  for (i=0; i < numBrands; i++) {
```

Comment:
Always use `var` when declaring your iterator in a for loop. Two reasons. First it creates a global variable, and second your code will fail when you are using strict.